### ISSUE TITLE
Incorrect adjective plain form?

### ISSUE THREAD

**Issue by cspotcode** [wkdonc/wkdonc.github.io#15]
This question confused me, is the answer incorrect, is that だ suffix erroneous?

It says it wants the plain form of the adjective, but then the answer has だ appended to the plain form.

I tried searching for the answer it expected on jisho, and it's interpreting the だ suffix as a separate word. https://jisho.org/search/%E6%84%8F%E5%9C%B0%E6%82%AA%E3%81%A0

![image](https://github.com/wkdonc/wkdonc.github.io/assets/376504/1b727560-28ac-4d33-b733-cfef8fef1e06)

**Comment by christopherball** [comment]
Here's my take on your question.  While い-adjectives have だ・です essentially built into them, should someone choose to add だ・です to their end, it's purely for politeness reasons.  In contrast, な-adjectives do not have だ・です built-in, and thus require them for conveying conjugated forms.  Here's a couple grammar guides touching on these points:

https://www.tofugu.com/japanese-grammar/na-adjective/
https://www.japanistry.com/adjectives/

**Comment by stmarier** [comment]
I agree with jisho that だ in this situation is a separate word.  It's true that you need it in a sentence for the sentence to be grammatically correct, but the prompt is not to construct a grammatically correct sentence, it's just to make the word "affirmative", which in this case is simply 意地悪.

**Comment by christopherball** [comment]
If it were to ask you to provide the past tense form, you'd say 意地悪だった.  Given this rationale, 意地悪だ seems like a reasonable expectation for the affirmative form.  Sure, you can leave だ off if you want to, but you're mixing levels of casualness if you do, as だ and じゃない are the same casual level, whereas completely leaving だ off would be even _more_ casual.

Edit: Figured I'd check what ChatGPT had to say on this matter and it appears to agree with my sentiment:
![image](https://github.com/wkdonc/wkdonc.github.io/assets/5790394/70168e99-c481-4095-83a3-541cdb4d7a79)

**Comment by cspotcode** [comment]
Thanks for looking at this!

Interestingly, ChatGPT says "omit だ at the end of the **sentence**" not "omit it from the **word**" which makes sense cuz it's a separate word.

And the Drill app tells me "the answer is the plain form of the **word**" even though it's actually multiple words.

What I didn't realize is that, *even in the negative form*, じゃない is a separate word, not part of the adjective. I didn't realize that the negative form presented by the quiz is actually multiple words.

At the end of the day my goal is to learn Japanese, not split hairs, so I *do* want to learn to include the extra word だ. But I *also* want to learn that it's a separate word, not a syllable of the adjective. I wonder if there's a reasonable way for the Drill's UI to teach me that when it explains the answer?

Does the quiz show the affirmative "意地悪 だ" (including da) and ask the user to convert to the negative "意地悪　じゃない"?

**Comment by cspotcode** [comment]
I just did a quiz with na adjectives and now I get it: they've always got that auxiliary verb and/or copula appended to them.  It's technically separate word(s).  Now that I realize the questions are always multiple words, it makes sense how the answers are multiple words. It's quizzing me on phrases, not words, because that's how na-adjectives work.

Thanks again for discussing this and helping me understand what's going on here.
